checkov/terraform/checks/resource/aws/PublicResourceBasedPolicy.py

```diff
@@ -0,0 +1,30 @@
+from typing import Dict, List, Any
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.common.util.type_forcers import force_list
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class PublicResourceBasedPolicy(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure resource-based policies don't allow public actions on resource"
+        id = "CKV_AWS_283"
+        supported_resources = ('aws_sns_topic_policy', 'aws_sqs_queue_policy', 'aws_s3_bucket_policy')
```

Comment:
we have this `CKV_AWS_70` for S3 policy and `CKV_AWS_72` for SQS policy and `CKV_AWS_169` for SNS policy

Reply:
Removing this check